sample/src/androidTest/kotlin/com/agoda/sample/screen/Pager2Screen.kt

```diff
@@ -0,0 +1,21 @@
+package com.agoda.sample.screen
+
+import android.view.View
+import com.agoda.kakao.pager2.KViewPager2
+import com.agoda.kakao.pager2.KViewPagerItem
+import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.text.KTextView
+import com.agoda.sample.R
+import org.hamcrest.Matcher
+
+open class Pager2Screen : Screen<Pager2Screen>() {
+    val pager: KViewPager2 = KViewPager2({
+                                             withId(R.id.pager)
+                                         }, itemTypeBuilder = {
+        itemType(Pager2Screen::SimpleItem)
+    })
+
+    class SimpleItem(parent: Matcher<View>) : KViewPagerItem<SimpleItem>(parent) {
+        val text: KTextView = KTextView { withId(R.id.text) }
```

Comment:
I think you need to pass the parent to the constructor. Is it possible to have 2 pages visible at the same time? If it is possible, you need to distinctively match the page's container.

Reply:
@Unlimity looks like it not possible. Only one Fragment at the same time. Don't see the reasons to pass parent to the constructor.

Reply:
If the API doesn't allow to show multiple pages, then we need to refactor item construction and remove parent matcher.

Reply:
On the same page, you can have items with the same IDs. F.E. ViewPager is 1\2 of the screen from the top and Second one just a FrameLayout with the same Fragment. I think that it's important to have a parent's matcher.

Reply:
Also, it will reflect that Item should be under the Pager hierarchy. I think that logically it looks much clear.

Reply:
You have a point here.